draft-ietf-tls-tls13.md

```diff
@@ -2139,9 +2203,9 @@ hash
   SHA-224, SHA-256, SHA-384, and SHA-512 {{SHS}}, respectively.  The
   "none" value is provided for future extensibility, in case of a
   signature algorithm which does not require hashing before signing.
-  The usage of MD5 and SHA-224 are deprecated. The md5_RESERVED and
-  sha224_RESERVED values MUST NOT be offered or negotiated by any
-  implementation.
+  The use of MD5, SHA-1, and SHA-224 are deprecated. The md5_RESERVED,
+  sha1_RESERVED, and sha224_RESERVED values MUST NOT be offered by
+  any implementation.
```

Comment:
You're right; there's a potential issue here. TLS 1.3 clients with this provision would offer SHA-256 and TLS 1.2 servers would either send a SHA-256 chain if they have one, send a chain with SHA-1 instead, or possibly abort. Based on recent discussion, yes, we would need to check what various implementations would do here. If they abort, I'd have to change this to allow sending SHA-1 support but not negotiating it for TLS 1.3.